### PR TITLE
46 cap deployment

### LINKED ISSUE #46: Create deployment routine using Capistrano
### What maintenance needs to be done?
We need a way to deploy updates to the application to staging and production machines. In order to be consistent with other projects, we should use Capistrano to deploy. 

### Level of urgency
- [x] High
- [ ] Moderate
- [ ] Low

### Why is this maintenance needed?
- This application has outgrown the use case for the libstatic infrastructure, and would benefit from being able to deploy independently from the other applications on libstatic.

### Acceptance criteria
 - [ ] You can deploy different branches to staging and production servers using Ansible Tower
 - [ ] You can deploy a specific using `BRANCH=mybranch bundle exec cap [environment] deploy` from your local environment
 - [ ] Build the dist directory. See [allsearch_frontend example](https://github.com/pulibrary/allsearch_frontend/blob/main/config/deploy.rb)

### Implementation notes, if any
- See [PR to add Capistrano to AllSearch Frontend](https://github.com/pulibrary/allsearch_frontend/pull/27) as an example

### PR DESCRIPTION
**Pull request by rladdusaw**
Resolves #46

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,8 +5,14 @@ Index of Marquand auction catalogs.
 ## Local development
 
 1. Run: `npm install`
-2. Start a local server: `npm start`
-3. Visit: [localhost:3456](http://localhost:3456/)
+2. Start a local server: `npm run dev`
+3. Visit: [localhost:3456](http://localhost:5173/)
```
Comment (maxkadel) [inline README.md:9]:
```suggestion
3. Visit: [localhost:5173](http://localhost:5173/)
```

File: app/services/DataService.js [hunk 2]
```diff
@@ -1,3 +1,5 @@
+import Papa from 'papaparse';
```
Comment (maxkadel) [inline app/services/DataService.js:1]: Do we still want this line?

File: README.md [hunk 3]
```diff
@@ -5,8 +5,14 @@ Index of Marquand auction catalogs.
 ## Local development
 
 1. Run: `npm install`
-2. Start a local server: `npm start`
-3. Visit: [localhost:3456](http://localhost:3456/)
+2. Start a local server: `npm run dev`
+3. Visit: [localhost:5173](http://localhost:5173/)
+
+## Building for deployment
```
Comment (maxkadel) [inline README.md:11]: So this is part of the capistrano deployment, so how can we make that clear? I think it's nice to have here, but I initially thought we needed to do them separately for deployment

**Review by maxkadel — CHANGES_REQUESTED** [review]
We lost the table description somewhere in the merges

**Comment by rladdusaw** [comment]
> We lost the table description somewhere in the merges

I'm seeing the [data attribute](https://github.com/pulibrary/static-tables/pull/72/files#diff-927334fc49930ff6860b35d0bc3943fc84d050b4f6b3b3b8e1d15a1e9c90f740L22) and it being used [in the template](https://github.com/pulibrary/static-tables/blob/ffd22b8185083ca33b4953cd8bc11941facdf86e/app/components/PaginatedTable.js#L174).

**Comment by maxkadel** [comment]
> > We lost the table description somewhere in the merges
> 
> I'm seeing the [data attribute](https://github.com/pulibrary/static-tables/pull/72/files#diff-927334fc49930ff6860b35d0bc3943fc84d050b4f6b3b3b8e1d15a1e9c90f740L22) and it being used [in the template](https://github.com/pulibrary/static-tables/blob/ffd22b8185083ca33b4953cd8bc11941facdf86e/app/components/PaginatedTable.js#L174).

But it's not displayed on the site when it's deployed:
Staging:
Staging site without table descirpiton

Production:
image

**Pull request merged**